app/forms/declaration/DepartureTransport.scala

```diff
@@ -65,6 +86,18 @@ object TransportCodes {
   val EuropeanVesselIDNumber = "80"
   val NameOfInlandWaterwayVessel = "81"
 
+  val extractBorderTransportValue: String => String = _ match {
```

Comment:
Similar, I'd prefer if we had an enum for this

Reply:
Removed